benchmarks/configs/defaults.py

```diff
@@ -0,0 +1,124 @@
+        # such as relative value changes.
+        "hsv": np.array([1, 0.5, 0.5]),
+    }
+}
+
+default_tolerance_values = {
+    "hsv": np.array([0.1, 0.2, 0.2]),
+    "principal_curvatures_log": np.ones(2),
+}
+
+default_tolerances = {
+    "patch": default_tolerance_values
+}  # features where weight is not specified default weight to 1
+
+default_evidence_lm_config = dict(
+    learning_module_class=EvidenceGraphLM,
+    learning_module_args=dict(
+        # mmd of 0.015 get higher performance but slower run time
+        max_match_distance=0.01,  # =1cm
+        tolerances=default_tolerances,
+        feature_weights=default_feature_weights,
+        # smaller threshold reduces runtime but also performance
+        x_percent_threshold=20,
+        # Using a smaller max_nneighbors (5 instead of 10) makes runtime faster,
+        # but reduces performance a bit
+        max_nneighbors=10,
+        # Use this to update all hypotheses at every step as previously
+        # evidence_update_threshold="all",
+        # Use this to update all hypotheses > x_percent_threshold (faster)
+        evidence_update_threshold="x_percent_threshold",
+        # use_multithreading=False,
+        # NOTE: Currently not used when loading pretrained graphs.
+        max_graph_size=0.3,  # 30cm
+        num_model_voxels_per_dim=100,
+        gsg_class=EvidenceGoalStateGenerator,
+        gsg_args=dict(
+            goal_tolerances=dict(
+                location=0.015,  # distance in meters
+            ),  # Tolerance(s) when determining goal-state success
+            elapsed_steps_factor=10,  # Factor that considers the number of elapsed
+            # steps as a possible condition for initiating a hypothesis-testing goal
+            # state; should be set to an integer reflecting a number of steps
+            min_post_goal_success_steps=5,  # Number of necessary steps for a hypothesis
+            # goal-state to be considered
+            x_percent_scale_factor=0.75,  # Scale x-percent threshold to decide
+            # when we should focus on pose rather than determining object ID; should
+            # be bounded between 0:1.0; "mod" for modifier
+            desired_object_distance=0.03,  # Distance from the object to the
+            # agent that is considered "close enough" to the object
+        ),
+    ),
+)
+
+default_evidence_1lm_config = dict(learning_module_0=default_evidence_lm_config)
+
+# NOTE: maybe lower once we have better policies
+# Is not really nescessary for good performance but makes sure we don't just overfit
+# on the first few points.
+min_eval_steps = 20
+
+monty_models_dir = os.getenv("MONTY_MODELS")
+
+# v6 : Using TLS for point-normal estimation
+# v7 : Updated for State class support + using new feature names like pose_vectors
+# v8 : Using separate graph per input channel
+# v9 : Using models trained on 14 unique rotations
+# v10 : Using models trained without the semantic sensor
+pretrained_dir = os.path.expanduser(
```

Comment:
I'm 50/50 about this and maybe this is the most practical spot, but not sure if this default is a bit out of place relative to the others, given the specific connection to YCB?